### PR TITLE
Added approving and removing issues.

### PR DESCRIPTION
**Pull request by f1nwe**
- [ ] @Kmalynss 
- [ ] @denys281 
- [ ] @TarasBardiuk

- Only admins and moderators can see list of all issues.
- Admins and moderators are able to approve or remove issues.

- Added controller Issues in account namespace
- Added rspecs for issues controller

Link to issue: https://github.com/Kotany-IF070-WebUI-RoR/NewVasjuky/issues/32

### CODE REVIEW

File: app/controllers/account/issues_controller.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+module Account
+  class IssuesController < ApplicationController
+    def index
+      @issues = Issue.page(params[:page]).per(20)
```
Comment (igorkasyanchuk) [inline app/controllers/account/issues_controller.rb]: add scope `ordered` in model, and return from newest to oldest

File: config/routes.rb [hunk 2]
```diff
@@ -11,6 +11,11 @@
   end
 
   namespace :account do
+    resources :issues, only: [:index] do
+      patch :approve
```
Comment (igorkasyanchuk) [inline config/routes.rb]: wrap it with 
```
member do
  patch :approve
...
end
```

File: app/policies/issue_policy.rb [hunk 3]
```diff
@@ -1,3 +1,22 @@
 # frozen_string_literal: true
 class IssuePolicy < ApplicationPolicy
+  def index?
```
Comment (igorkasyanchuk) [inline app/policies/issue_policy.rb]: from today do not add new policies.
add in controller
before_action :require_admin_and_moderator

(I was saying about it today in the evening)

File: app/controllers/account/issues_controller.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      authorize @issues
+    end
+
+    def approve
+      @issue = Issue.find_by_id(params[:issue_id])
```
Comment (igorkasyanchuk) [inline app/controllers/account/issues_controller.rb]: change to Issue.find(params[:id]) + made changes in routes file as I said

File: app/controllers/account/issues_controller.rb [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+      authorize @issue
+      @issue.update_attribute('approved', 'true')
+      redirect_to account_issues_path
+    end
+
+    def destroy
+      @issue = Issue.find_by_id(params[:issue_id])
```
Comment (igorkasyanchuk) [inline app/controllers/account/issues_controller.rb]: same here

File: app/views/account/issues/index.html.slim [hunk 6]
```diff
@@ -0,0 +1,15 @@
+h1 Issues
+= link_to 'Home', root_path
+.container
```
Comment (igorkasyanchuk) [inline app/views/account/issues/index.html.slim]: use correct bootstrap styling

```
.row
   .col-md-12
      h1 Issues
      table ....
```

everything must be in ROW and COLS

File: app/views/account/users/index.html.slim [hunk 7]
```diff
@@ -1,4 +1,5 @@
 h1 All Users
+= link_to 'Home', root_path
```
Comment (igorkasyanchuk) [inline app/views/account/users/index.html.slim]: same here

File: app/policies/issue_policy.rb (hunk 3 above)
Comment (f1nwe) [inline app/policies/issue_policy.rb]: We will fix it after demo on Thursday.

File: app/controllers/account/issues_controller.rb [hunk 8]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+module Account
+  class IssuesController < ApplicationController
+    def index
+      @issues = Issue.ordered.page(params[:page]).per(20)
+      authorize @issues
+    end
+
+    def approve
+      @issue = Issue.find_by_id(params[:id])
```
Comment (igorkasyanchuk) [inline app/controllers/account/issues_controller.rb]: change to `find` and everywhere too

File: app/views/account/issues/index.html.slim [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+          thead
+            tr
+              th Issue ID
+              th Name
+              th Status
+              th Remove
+          tbody
+            tr
+              = render @issues
+    = paginate @issues, theme: 'twitter-bootstrap-3'
```
Comment (igorkasyanchuk) [inline app/views/account/issues/index.html.slim]: pagination must be on same level as table (inside .container)

File: app/views/account/users/index.html.slim [hunk 10]
```diff
@@ -1,14 +1,17 @@
...
+          thead
+            tr
+              th Email
+              th Name
+              th Ban
+              th Role
+          tbody
+            tr
+              = render @users
+    = paginate @users, theme: 'twitter-bootstrap-3'
```
Comment (igorkasyanchuk) [inline app/views/account/users/index.html.slim]: same here

File: config/routes.rb [hunk 11]
```diff
@@ -5,12 +5,17 @@
...
-    post 'wp-admin', to: 'devise/sessions#create', as: :user_session
-    delete 'logout', to: 'devise/sessions#destroy', as: :destroy_user_session
+    get 'sign-in', to: 'devise/sessions#new', as: :new_user_session
   end
 
   namespace :account do
+    resources :issues, only: [:index] do
+      member do
+        patch :approve
+        delete :destroy
```
Comment (igorkasyanchuk) [inline config/routes.rb]: this action is by default provided by REST, remove it from routes, but add to "only" (above)

File: db/seeds.rb [hunk 12]
```diff
@@ -1,17 +1,26 @@
...
-                                                    banned: true)
+User.where(email: 'admin@test.com').first_or_create(password: 'password',
+                                                   role: :admin)
+User.where(email: 'moderator@test.com').first_or_create(password: 'password',
+                                                    role: :moderator)
+User.where(email: 'reporter@test.com').first_or_create(password: 'password',
+                                                   role: :reporter)
+
+
+99.times do |n|
```
Comment (igorkasyanchuk) [inline db/seeds.rb]: remove this. Soon we will have a good seed for issues.

**Pull request merged**